### PR TITLE
update eco list July 3

### LINKED ISSUE #64: Suggest a project - Mest
### Project

Mest

### Description

Mest is an open, user-friendly platform for sharing and labeling blockchain transactions and wallet collections. It aims to enrich the blockchain community by allowing users to contribute content that serves the public good. Mest aspires to be a central hub for valuable blockchain insights and a collective effort to navigate the blockchain landscape.

### Please provide the project scope.

Safe Apps

### Select a primary category of your project.

DAO Tooling

### Select secondary categories of your project.

Tooling

### Project logo

MestLogo

### Value proposition

Mest's value proposition for Safe revolves around enhancing the visibility and understanding of transactions performed through Gnosis Safe. The core issue addressed by Mest is the difficulty in understanding transaction details due to the way Gnosis Safe executes transactions, often making it hard to discern the actual actions taken and the asset changes involved.

Mest offers a solution by parsing the execTransaction method calls within Gnosis Safe contracts to extract actionable data, counterparties, and even signer information from signatures. This allows for a deeper analysis and transformation of transaction data into human-readable forms, covering a wide range of scenarios like DEX swaps, NFT trades, bridging, lending, and more. Unlike other platforms that may not correctly interpret Safe transactions, leading to a lack of clarity on what was done and which assets were affected, Mest aims to provide a comprehensive breakdown.

Key improvements Mest brings to the table include:

- The ability to understand who the counterparties in a transaction are, even when they are Gnosis Safe wallets.
- Enhanced readability by parsing and displaying the actual method called instead of a generic execTransaction.
- Deeper transaction analysis that covers various scenarios beyond basic transfers, such as trades, liquidity actions, and more.
- Richer counterparty metadata, including logos and identities, to better understand transaction contexts.
- Accurate asset change analysis, as opposed to missing or incorrect information on other platforms.

### Contact

lucas@mest.io

### Project URL

https://mest.io/

### Github / Dev Docs

https://github.com/mest-io

### Twitter

mest_io

### Primary integration

Safe Apps SDK

### Packages

Safe Core SDK

### Modules / Guards

_No response_

### Smart Accounts

_No response_

### Networks

Base, Ethereum, BNB, Gnosis Chain, Polygon, Optimism, Avalanche, Arbitrum, Aurora

### Can you create a Safe via the interface?

No

### Can you import an existing Safe via the interface?

No

### LINKED ISSUE #82: Satoshi Safe
### Project

Satoshi Safe

### Description

Secure multi-sig crypto wallet for high net-worth users, hedge funds, and institutions.

### Please provide the project scope.

Core SDK Integration

### Select a primary category of your project.

Infrastructure

### Select secondary categories of your project.

DAO Tooling, Institutional, Wallet

### Project logo

Screenshot 2024-04-29 at 12 05 52 PM

### Value proposition

Satoshi Safe extends Gnosis Safe by offering a streamlined, intuitive interface that simplifies transaction management for high net-worth individuals, family offices, and hedge funds. Our platform integrates enhanced security features, robust support, and cross-chain capabilities, making it the premier choice for sophisticated investors seeking a reliable and efficient multi-signature wallet solution.

### Contact

matt@getsatoshisafe.com

### Project URL

https://www.getsatoshisafe.com

### Github / Dev Docs

https://github.com/satoshisafedevs/multisig

### Twitter

@getsatoshisafe

### Primary integration

Core SDK, Transaction Service API

### Packages

Safe Core SDK, Safe Service Client, Transaction Service API

### Modules / Guards

        "@safe-global/api-kit": "^1.3.0",
        "@safe-global/protocol-kit": "^1.2.0"

### Smart Accounts

To leverage the benefits of Smart Accounts effectively, Satoshi Safe offers fund managers robust multisig functionality. Our platform allows for customizable multi-signature settings that require multiple signers to approve transactions, thereby enhancing security and reducing the risk of unauthorized access or fraud. This feature is crucial for high net-worth individuals, family offices, and hedge funds, ensuring that large transactions undergo rigorous scrutiny before execution.

### Is Safe the only Smart Account provider you are using in your project?

No

### Networks

Arbitrum, Avalanche, Base, BNB, Ethereum, Optimism, Polygon PoS

### Can you create a Safe via the interface?

Yes

### Can you import an existing Safe via the interface?

Yes

### LINKED ISSUE #87: Palmera
### Project

Palmera

### Description

Smart Treasury Management: Multiple Safes, Multiple Chains, One Dashboard.

### Please provide the project scope.

Safe Apps

### Select a primary category of your project.

Payments

### Select secondary categories of your project.

Dashboard, DAO Tooling, Accounting, Aggregator

### Project logo

Palmera Logo

### Value proposition

Palmera aggregates all your Safes across multiple chains to streamline your operations and treasury management, all from a single dashboard.

### Contact

general@palmeradao.xyz

### Project URL

https://www.palmeradao.xyz/

### Github / Dev Docs

_No response_

### Twitter

https://x.com/palmera_dao

### Primary integration

Core SDK, Transaction Service API, Safe Contracts

### Packages

Safe Core SDK, Safe Service Client, Transaction Service API, Safe Deployments, Safe Ethers Adapter, web3-onboard

### Modules / Guards

_No response_

### Smart Accounts

_No response_

### Is Safe the only Smart Account provider you are using in your project?

Yes

### Networks

Arbitrum, Base, Celo, Ethereum, Gnosis Chain, Optimism, Polygon PoS

### Can you create a Safe via the interface?

Yes

### Can you import an existing Safe via the interface?

Yes

### LINKED ISSUE #89: Protofire DAO
### Project

Protofire DAO

### Description

Reliable partner for Safe and DeFi deployment, customization & development, supporting 50+ networks

### Please provide the project scope.

Network

### Select a primary category of your project.

Infrastructure

### Select secondary categories of your project.

DeFi, SAFE, Infrastructure

### Project logo

proto

### Value proposition

_No response_

### Contact

ivan@protofire.io

### Project URL

https://protofire.io/services/safe-deployment

### Github / Dev Docs

https://github.com/protofire

### Twitter

https://twitter.com/protofire

### Primary integration

Safe Apps SDK, Core SDK, Transaction Service API, Safe Contracts

### Packages

Safe Core SDK, Safe Service Client, Transaction Service API, Safe Deployments, Safe Ethers Lib, Safe Ethers Adapter, web3-onboard, web3-react, safe-apps-wagmi, safe-apps-web3modal

### Modules / Guards

_No response_

### Smart Accounts

_No response_

### Is Safe the only Smart Account provider you are using in your project?

Yes

### Networks

Blast, Optimism, Scroll

### Can you create a Safe via the interface?

Yes

### Can you import an existing Safe via the interface?

Yes

### LINKED ISSUE #92: Suggest a project: pier wallet
### Project

pier wallet

### Description

pier wallet is a **Wallet as a Service** (WaaS) provider from Switzerland, leveraging **account abstraction** to offer seamless (up to invisible) wallet experiences. Besides our own infrastructure, we leverage **Safe** technology for **multi-signature** wallets. 

### Please provide the project scope.

Core SDK Integration

### Select a primary category of your project.

Infrastructure

### Select secondary categories of your project.

Governance, Dashboard, Institutional, Accounting

### Project logo

pier_logo-standard_black

### Value proposition

_No response_

### Contact

mario@pierwallet.com

### Project URL

https://www.pierwallet.com/

### Github / Dev Docs

https://github.com/pier-wallet

### Twitter

https://twitter.com/pier_wallet

### Primary integration

Transaction Service API, Safe Contracts

### Packages

Safe Core SDK, Safe Ethers Lib, Safe Ethers Adapter, web3-react

### Modules / Guards

_No response_

### Smart Accounts

_No response_

### Is Safe the only Smart Account provider you are using in your project?

No

### Networks

Arbitrum, Base, Blast, Ethereum, Optimism, Polygon PoS

### Can you create a Safe via the interface?

Yes

### Can you import an existing Safe via the interface?

No

### LINKED ISSUE #93: Tezoro
### Project

Tezoro

### Description

Make your crypto recoverable

### Please provide the project scope.

Safe Apps

### Select a primary category of your project.

DeFi

### Select secondary categories of your project.

DeFi, Insurance, Automation

### Project logo

Tezoro 500х500 black

### Value proposition

Create a smart contract that can recover your crypto if something goes wrong. 

### Contact

hello@tezoro.io

### Project URL

https://tezoro.io/

### Github / Dev Docs

https://gitlab.com/tezoro/tezoro-frontend-2.0

### Twitter

https://twitter.com/tezoroio

### Primary integration

Safe Apps SDK

### Packages

safe-apps-wagmi

### Modules / Guards

To ensure the security of the service, Tezoro uses a long list of technologies and libraries at the smart contract, front-end and back-end levels. For example, Viem and Wagmi. Tezoro was audited by Consensys Diligence and CertiK.

### Smart Accounts

Smart Accounts improve the safety of funds and the security of recovery and inheritance transactions.

### Is Safe the only Smart Account provider you are using in your project?

Yes

### Networks

Ethereum

### Can you create a Safe via the interface?

No

### Can you import an existing Safe via the interface?

Yes

### PR DESCRIPTION
**Pull request by zzuziak**
ECO DB cleanup &:
- fixes #64
- fixes #93
- fixes #92
- fixes #89 
- fixes #82 
- fixes #87

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1888,6 +1854,23 @@
...
...\nMest offers a solution by parsing the execTransaction method calls within Gnosis Safe contracts to extract actionable data, counterparties, and even signer information from signatures. This allows for a deeper analysis and transformation of transaction data into human-readable forms, covering a wide range of scenarios like DEX swaps, NFT trades, bridging, lending, and more. Unlike other platforms that may not correctly interpret Safe transactions, leading to a lack of clarity on what was done and which assets were affected, Mest aims to provide a comprehensive breakdown.\nKey improvements Mest brings to the table include:\n• The ability to understand who the counterparties in a transaction are, even when they are Gnosis Safe wallets.\n• Enhanced readability by parsing and displaying the actual method called instead of a generic execTransaction.\n• Deeper transaction analysis that covers various scenarios beyond basic transfers, such as trades, liquidity actions, and more.\n• Richer counterparty metadata, including logos and identities, to better understand transaction contexts.\n• Accurate asset change analysis, as opposed to missing or incorrect information on other platforms.",
```
Comment (DiogoSoaress) [inline data.json:1864]: Quite big description 😅

File: logos/Protofirelogo192.png
Comment (DiogoSoaress) [inline logos/Protofirelogo192.png:1]: Observation: I think it is neater to adhere to simpler logo names. I would rename this one simply `protofire.png`.

File: data.json [hunk 2]
```diff
@@ -2517,6 +2483,23 @@
     "safe_apps_smart": "",
     "networks": "Polygon"
   },
+  {
+    "project": "pier wallet",
```
Comment (DiogoSoaress) [inline data.json:2487]: Non capitalized on purpose?

**Review by DiogoSoaress — COMMENTED** [review]
Couple of comments but optional to address them.

You can remove from the /logos folder the logos from the removed projects.

File: data.json (hunk 2 above)
Comment (zzuziak) [reply data.json:2487]: yup, that's part of the branding

File: logos/Protofirelogo192.png
Comment (zzuziak) [reply logos/Protofirelogo192.png:1]: We can take care of that with the transition to using Attio as the source of truth

**Review by DiogoSoaress — APPROVED** [review]
Looks good now ✅

**Pull request merged**